### ISSUE TITLE
Crash managing bookmarks folders

### ISSUE THREAD

**Issue by Jonzuk** [bromite/bromite#2140]
### Preliminary checklist

- [x] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

102.0.5005.96

### Device architecture

arm64

### Android version

12.0

### Device model

OnePlus 7 Pro

### Changed flags

Everything is default 

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

https://ufile.io/8pbdz6p4 i cant copy it all here is the file txt

### Describe the bug

Everytime you press change parent folder location for bookmarks ot crashes

### Steps to reproduce the bug

1.Press Bookmarks on Homepage
2. Select mobile bookmarks
3. Select the 3 dots at the end of a bookmark
4. Press Edit
5. Press the folder 
6. New folder
7. Change new folder parent location and crash when you select it

### Expected behavior

Immediately crashes and the phone says the app is no longer responding after you rake those steps

**Comment by IconBunnyGoing** [comment]
Can confirm. Bromite seems to crash when changing parent folder of a new folder. However, I observed a different behaviour. My screen turns white, then the browser seems to restart in the state before trying to create a new folder. However, the behaviour seems inconsistent, as the screen sometimes displays a different bookmark than the one I was editing, or it sometimes displays "Managed Bookmark" or something akin to it.

**Comment by uazo** [comment]
I confirm the crash:

```
06-19 14:29:05.279  5078  5078 E AndroidRuntime: FATAL EXCEPTION: main
06-19 14:29:05.279  5078  5078 E AndroidRuntime: Process: org.bromite.bromite.dev, PID: 5078
06-19 14:29:05.279  5078  5078 E AndroidRuntime: java.lang.AssertionError
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at org.chromium.chrome.browser.bookmarks.BookmarkBridge.getBookmarkById(BookmarkBridge.java:514)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at org.chromium.chrome.browser.bookmarks.BookmarkEditActivity.onCreate(BookmarkEditActivity.java:68)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7136)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7127)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2893)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3048)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1808)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:193)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6669)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
06-19 14:29:05.279  5078  5078 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

at [this line](https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkBridge.java;l=469;drc=256182eb99a7e9de19375dc250df8b3d79e0bda8?q=BookmarkBridge.java&ss=chromium%2Fchromium%2Fsrc)

**Comment by uazo** [comment]
it was probably an upstream error, because it will be fixed with 103

**Comment by csagan5** [comment]
Fixed in `103.0.5060.121`.

Please report back if not yet fixed.
